NataliiaZinchenko/HW7/nzinchenko_7.2.py

```diff
@@ -0,0 +1,88 @@
+# Writing program which calculate area of a rectangle, triangle or circle
+
+
+def calculate_rectangle_area(length, width):
+    '''Function that calculates rectangle area using length and width
+       length: int OR float
+       width: int OR float
+    '''
+    rectangle_area = length * width
+    return rectangle_area
+
+
+def calculate_rectangle_area_console():
+    '''Function that printing out the result of rectangle area calculation,
+       depending on the data type of rectangle's length & width (int / float)
+    '''
+    print("Ok! Let's calculate rectangle area!\n")
+    custom_rectangle_length = float(input("Enter rectangle length: \n"))
+    custom_rectangle_width = float(input("Enter rectangle width: \n"))
+
+
+    output_rectangle_area = calculate_rectangle_area(custom_rectangle_length, custom_rectangle_width)
```

Comment:
In this case I recommend  don't use second function
return  custom_rectangle_length*custom_rectangle_width